src/System.Runtime.Numerics/src/System/Numerics/BigCalc.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace System.Numerics
+{
```

Comment:
The plan was to categorize all those algorithms, so we do not have one huge code file. But I'll remove it, don't know if needed anyway.